src/translations/pages/assistant.ts

```diff
@@ -13,6 +13,18 @@ export const Assistant = {
       ),
       from: _('Fra', 'From', 'Frå'),
       to: _('Til', 'To', 'Til'),
+      placholder: {
+        from: _(
+          'adresse, kai eller holdeplass',
+          'address, pier or bus stop',
+          'adresse, kai eller haldeplass',
+        ),
+        to: _(
+          'ankomstdestinasjon',
+          'arrival destination',
+          'ankomstdestinasjon',
```

Comment:
Is this modelled from the app? I think this is an example of lack of clear speech in a way. Why is it "ankomstdestinasjon" is that something that people actually say? Have you ever asked anyone "ja, hva er ankomstdestinasjonen din"? 😅 Maybe "ankomststed" is better? Or just the same as from?